File: Assignments/Assignment_17/p8.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                for(int j=1;j<=i;j++){

                        if(i%3==0){

                                if(j%2==0){

                                        printf("%c\t",(64+j));
                                }else{

                                        printf("%d\t",j);
                                }
                        }else{

                                printf("%d\t",j);
                        }
                }printf("\n\n");
        }
}

File: JAVA/11.Triangle_Pattern/p8.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
		int count1=1,count2=1;
		for(int i=row;i>0;i--){
		
			count2=count1;
			for(int j=0;j<i;j++){
			
					System.out.print(count2+"\t");
					count2++;
			}System.out.println();
			count1++;
		}
	}
}
